### PR TITLE
Update PO store for updated protos

### PR DESCRIPTION
**Pull request by dnewh**
This updates the PO store to account for the changes to the PO protobuf messages. This updates the schemas, DB models, Grid structs, DB operations, and REST API to account for the different fields in the updated messages.

### CODE REVIEW

File: sdk/src/migrations/diesel/postgres/migrations/2021-09-10-131000_update_purchase_order_tables/down.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- -----------------------------------------------------------------------------
+
+ALTER TABLE purchase_order
+RENAME COLUMN purchase_order_uid TO uuid,
+DROP COLUMN buyer_org_id VARCHAR(256) NOT NULL,
+DROP COLUMN seller_org_id VARCHAR(256) NOT NULL,
```
Comment (shannynalayna) [inline sdk/src/migrations/diesel/postgres/migrations/2021-09-10-131000_update_purchase_order_tables/down.sql:19]: I think the last comma here (on the last line before the final one) is causing an error for the tests. Need to remove it from this line and I believe line 23

File: sdk/src/migrations/diesel/postgres/migrations/2021-09-10-131000_update_purchase_order_tables/up.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- -----------------------------------------------------------------------------
+
+ALTER TABLE purchase_order
+RENAME COLUMN uuid TO purchase_order_uid,
+DROP COLUMN org_id,
+ADD COLUMN buyer_org_id VARCHAR(256) NOT NULL,
```
Comment (shannynalayna) [inline sdk/src/migrations/diesel/postgres/migrations/2021-09-10-131000_update_purchase_order_tables/up.sql:19]: Same sitch here, I believe this is causing the error with the ci tests

File: sdk/src/purchase_order/store/diesel/mod.rs [hunk 3]
```diff
@@ -70,7 +70,8 @@ impl PurchaseOrderStore for DieselPurchaseOrderStore<diesel::pg::PgConnection> {
 
     fn list_purchase_orders(
         &self,
-        org_id: Option<String>,
+        buyer_org: Option<String>,
```
Comment (shannynalayna) [inline sdk/src/purchase_order/store/diesel/mod.rs:73]: Please update the `buyer_org` -> `buyer_org_id` and `seller_org` -> `seller_org_id`

File: sdk/src/migrations/diesel/postgres/migrations/2021-09-10-131000_update_purchase_order_tables/down.sql [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+ALTER TABLE purchase_order RENAME COLUMN purchase_order_uid TO uuid;
+ALTER TABLE purchase_order DROP COLUMN buyer_org_id;
+ALTER TABLE purchase_order DROP COLUMN seller_org_id;
+ALTER TABLE purchase_order ADD COLUMN org_id VARCHAR(256) NOT NULL;
+
+ALTER TABLE purchase_order_version RENAME COLUMN purchase_order_uid TO purchase_order_uuid;
+ALTER TABLE purchase_order_version ADD COLUMN org_id VARCHAR(256) NOT NULL;
+
+ALTER TABLE purchase_order_version_revision ADD COLUMN org_id VARCHAR(256) NOT NULL;
```
Comment (shannynalayna) [inline sdk/src/migrations/diesel/postgres/migrations/2021-09-10-131000_update_purchase_order_tables/down.sql:24]: I'm kind of confused why the down.sql file here is adding the `org_id` column back ?

Comment (dnewh) [reply sdk/src/migrations/diesel/postgres/migrations/2021-09-10-131000_update_purchase_order_tables/down.sql:24]: The down file is for essentially reverting the up file unless I'm mistaken

**Review by dplumb94 — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**